spec/controllers/admin/videos_controller_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'spec_helper'
+
+describe Admin::VideosController do 
+  describe "GET new" do
+    it_behaves_like "requires sign in" do
+      let(:action) {get :new}
+    end
+    it "sets the @video to a new video" do
+      set_current_admin
+      get :new
+      expect(assigns(:video)).to be_instance_of Video
+      expect(assigns(:video)).to be_new_record
```

Comment:
[`be_a_new`](https://www.relishapp.com/rspec/rspec-rails/v/2-1/docs/matchers/be-a-new-matcher)

> The be_a_new matcher accepts a class and passes if the subject is an instance
> of that class that returns true to new_record?

`expect(assigns(:video)).to be_a_new Video`